IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -267,51 +397,56 @@ def describe_screen(screen_id):
 screen_details_df.to_csv(output_file, index=False, sep="\t")
 
 
-# In[13]:
+# In the chunk below, we collect and process a subset of the available IDR screens. The subset contains IDR accession names idr0080, idr0001, and idr0069. The idr0080 dataset was collected by Way et al. (2021) and will be used as a validation set for our statistics pipeline as we already know the image counts and unique attribute counts. The other two datasets are used for prototyping iterative functionalities of metadata collection and statistics.
+
+# In[14]:
+
 
+# Collect study_names and imaging method metadata for each screen
+idr_names_dict = dict()
+screen_ids = [
+    screen_details_df.at[index, "screen_id"]
+    for index in range(len(screen_details_df.index))
+]
+idr_names = [
+    screen_details_df.at[index, "idr_name"]
+    for index in range(len(screen_details_df.index))
+]
+img_types = [
+    screen_details_df.at[index, "Imaging Method"]
+    for index in range(len(screen_details_df.index))
+]
+samples = [
+    screen_details_df.at[index, "Sample Type"]
+    for index in range(len(screen_details_df.index))
+]
+
+metadata = list(zip(screen_ids, img_types, samples))
+for name, metadata in zip(idr_names, metadata):
+    idr_names_dict[name] = metadata
+
+# Subset of studies for prototyping
+study_subset = [
+    "idr0080-way-perturbation/screenA",
+    "idr0001-graml-sysgro/screenA",
+    "idr0069-caldera-perturbome/screenA",
+]
+subset_metadata = [idr_names_dict[study] for study in study_subset]
+
+# Build the iterative object for pool.starmap()
+test_list = list(zip(study_subset, subset_metadata))
```

Comment:
Can you name `test_list` something more descriptive?

Maybe `prototype_idr_studies`?

Reply:
The `test_list` variable was changed to `prototype_idr_studies`.

Added in 543604f39f06022f944be050993f7f50d9d090b6.